index.html

```diff
@@ -29,5 +31,115 @@ <h3> I'm a software developer </h3>
         </ul>
       </div>     
   </section>
+  <section class="works">
+    <div class="recent-works"> 
+      <h1> My recent works </h1>
+    </div>
+    <img src="img/Placeholder.svg" alt="">
```

Comment:
- [ ] Your image here is not aligned with the designed. Kindly adjust to match design guidelines in terms of height and width.
### Your Design
![image](https://user-images.githubusercontent.com/43995830/153609706-280d4f29-6f41-4373-a7ed-783be5730b63.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/153609831-a87f8faf-6ba0-43ef-8ec4-6f4b82f0f577.png)